docs/traffic-control.md

```diff
@@ -0,0 +1,239 @@
+that hosts the Pod. The port doesn't need to exist in advance, Antrea will
+create the port if it doesn't exist. To use an OVS internal port, the `name` of
+the port must be provided:
+
+```yaml
+ovsInternal:
+  name: tap0
+```
+
+**device**: This specifies a network device on all Nodes. A Pod's traffic will
+be redirected or mirrored to the network device on the same Node that hosts the
+Pod. The network device must exist on all Nodes and Antrea will attach it to the
+OVS bridge if not already attached. To use a network device, the `name` of the
+device must be provided:
+
+```yaml
+device:
+  name: eno2
+```
+
+**geneve**: This specifies a remote destination for a GENEVE tunnel. All
+selected Pods' traffic will be redirected or mirrored to the destination via
+a GENEVE tunnel. The `remoteIP` field must be provided to specify the IP address
+of the destination. Optionally, the `destinationPort` field could be used to
+specify the UDP destination port of the tunnel, or 6081 will be used by default.
+If Virtual Network Identifier (VNI) is desired, the `vni` field can be specified
+to an integer in the range 0-16,777,215:
+
+```yaml
+geneve:
+  remoteIP: 10.0.10.2
+  destinationPort: 6081
+  vni: 1
+```
+
+**vxlan**: This specifies a remote destination for a VXLAN tunnel. All
+selected Pods' traffic will be redirected or mirrored to the destination via
+a VXLAN tunnel. The `remoteIP` field must be provided to specify the IP address
+of the destination. Optionally, the `destinationPort` field could be used to
+specify the UDP destination port of the tunnel, or 4789 will be used by default.
+If Virtual Network Identifier (VNI) is desired, the `vni` field can be specified
+to an integer in the range 0-16,777,215:
+
+```yaml
+vxlan:
+  remoteIP: 10.0.10.2
+  destinationPort: 4789
+  vni: 1
+```
+
+**gre**: This specifies a remote destination for a GRE tunnel. All selected
+Pods' traffic will be redirected or mirrored to the destination via a GRE
+tunnel. The `remoteIP` field must be provided to specify the IP address of the
+destination. If GRE key is desired, the `key` field can be specified to an
+integer in the range 0-4,294,967,295:
+
+```yaml
+gre:
+  remoteIP: 10.0.10.2
+  key: 1
+```
+
+### ReturnPort
+
+The `returnPort` field should only be set when the `action` is `Redirect`. It is
+similar to the `targetPort` field, but meant for specifying the port from which
+the traffic will be sent back to OVS and be forwarded to its original
+destination.
+
+## Examples
+
+### Mirroring all traffic to remote analyzer
+
+In this example, we will mirror all Pods' traffic and send them to a remote
+destination via a GENEVE tunnel:
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha2
+kind: TrafficControl
+metadata:
+  name: mirror-all-to-remote
+spec:
+  appliedTo:
+    podSelector: {}
+  direction: Both
+  action: Mirror
+  targetPort:
+    geneve:
+      remoteIP: 10.0.10.2
+```
+
+### Redirecting specific traffic to local analyzer
```

Comment:
Should we just use "local receiver" to be generic? Typically, redirection is not for an analyzer that does not change/drop the packets.